test/modules/home/home_view_test.dart

```diff
@@ -238,5 +244,54 @@ void main() {
         expect(delegate.showDetailButtonDidTap, emits(surveys.first));
       });
     });
+
+    describe("it's showSideMenu() is called", () {
+      beforeEach((tester) async {
+        await tester.pumpAndSettle();
+        module.view.showSideMenu();
+        await tester.pumpAndSettle();
+      });
+
+      it("triggers delegate's sideMenuDidShow emits", (tester) async {
+        expect(delegate.sideMenuDidShow, emits(null));
+      });
+
+      describe("then wipes from left to right", () {
+        beforeEach((tester) async {
+          final location = tester.getCenter(find.byKey(HomeView.bodyKey));
+          await tester.flingFrom(location, Offset(location.dx, 0), location.dx);
+          await tester.pumpAndSettle();
+        });
+
+        it("triggers delegate's sideMenuDidDismiss emits", (tester) async {
+          expect(delegate.sideMenuDidDismiss, emits(null));
+        });
+      });
+    });
+
+    describe("it's user avatar button is tapped", () {
```

Comment:
same here, the grammar structure is not very consistent you might need to recheck them all 🙏